### ISSUE TITLE
edge case accessing owned artifact

### ISSUE THREAD

**Issue by graybeal** [metadatacenter/cedar-project#1144]
An artifact at https://cedar.metadatacenter.org/instances/edit/https://repo.metadatacenter.org/template-instances/4e9cf0c8-8504-4ef8-a7a8-bc3760b785b6 is owned by one person, but lives in another person's directory. The owner can't see the artifact. The artifact can be found with this search pattern: 
 "Title of the project":"disability using ICT" AND "What assets":

(This happened when the original owner changed the owner to the new owner.)

**Comment by graybeal** [comment]
We fixed this in the short term (we hope) by changing ownership back, and giving write privileges to the second person.

Things to try in a test case:
- can we duplicate it? if so...
- it is accessible to the new owner via a direct URL to edit?
- can it be found by the new owner with a search?
That may give a clue whether it's owner permissions or search indexing...

**Comment by graybeal** [comment]
To test, I create an instance as CEDAR Admin, then used Sharing to change owner to me. 

I confirmed that the instance could not be viewed in my Workspace, nor in my Shared with Me (or Shared with Everybody). A search for it did not find it. But I was able to edit it directly using the edit IRI for it.

Moving it elsewhere does not make it more visible in my Workspace; it has to be in my directory to be shown in my Workspace. (Logical in one sense, I guess.)

No matter where I move it, even in my own Workspace, it does not show up in a search done by me. (At least so far. See below.)

If I reverse the ownership back to owned by CEDAR Admin, and make it writable by me, now it is visible in shared with me. But the search still doesn't find it. So possibly updating the search indexes can take a long time?
